engine/servermaster/jobmanager_test.go

```diff
@@ -664,3 +630,40 @@ func TestOnWorkerDispatchedFastFail(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, meta, 1)
 }
+
+func TestJobOperatorBgLoop(t *testing.T) {
+	t.Parallel()
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	masterID := "job-operator-bg-loop-test"
+	mockMaster, mgr := prepareMockJobManager(ctx, t, masterID)
+	mockMaster.On("InitImpl", mock.Anything).Return(nil)
```

Comment:
```suggestion
```

Reply:
Ref: https://github.com/pingcap/tiflow/pull/7072/files#diff-853e09825aa9d13f03821b40301ca876e7ad6b3c1e40badca6c48794d7220df4R54

Reply:
Removed this line from `prepareMockJobManager`, since mock functions should be customized outside.